core/mo-services-xml/src/main/resources/xml/ServiceDefPLATFORM.xml

```diff
@@ -1285,6 +1285,12 @@
                 <mal:type area="Platform" list="false"
                           name="AttitudeMode" service="AutonomousADCS" />
               </mal:field>
+              <mal:field canBeNull="true"
```

Comment:
As far as I recall MAL disallows adding anything in the message after an abstract field.
Moreover, why add it at the top-level of the message? I suggest adding this field to AttitudeMode abstract composite.

Reply:
attitudeDeterminationMode added as <mal:field></mal:field> to the AttitudeMode structure.